.github/workflows/test.yml

```diff
@@ -27,16 +27,5 @@ jobs:
     - name: Install dependencies
       run: npm install
 
-    - name: Build project
```

Comment:
I like that you've removed the coverage report generation because I actually never got CODECOV setup. But please don't remove the build step. We want to see that the build succeeds in CI after all.